BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -296,7 +296,8 @@ - (NSArray *)fullFileFieldsArray
                        BOXAPIObjectKeyExtension,
                        BOXAPIObjectKeyIsPackage,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
-                       BOXAPIObjectKeyCollections];
+                       BOXAPIObjectKeyCollections,
+                       BOXAPIObjectKeyRepresentations];
```

Comment:
do you know how performant the request for representations is? just wonder if we should exclude this field in the app, and just have it used by preview sdk.